docs/dockercompose/7-snmpv3-secrets.md

```diff
@@ -1,85 +1,89 @@
 # SNMPv3 secrets
 
-Creating a secret requires updating configuration of several docker compose files. To simplify this process, inside the 
-`docker_compose` package there is a `manage_secrets.py` file which will automatically manage secrets.
+## Migration steps
+Managing SNMPv3 secrets previously required updating docker compose files using the manage_secrets.py script.
+From SC4SNMP 1.15.0, this process has been simplified and can manage all SNMPv3 secrets using a single secrets.json file.
+
+#### 1. For setups not yet migrated to latest version
+First, delete all existing secrets from docker-compose.yaml using manage_secrets.py with the following flags:
```

Comment:
will it fail if the user won't do it? I'm afraid nobody's reading the documentation 😂

Reply:
ok and if somebody migrate to the newest version (both package and images) they need to go through the process of secret creation once again?